openkfw/TruBudget
# frontend:Add missing translation

### Checklist




### Description


### How to test

1. Open permission popup window and check if all translations are there




Closes #1889
